src/SFA.DAS.Reservations.Application.UnitTests/Employers/Services/WhenGettingTrustedEmployers.cs

```diff
@@ -46,17 +44,15 @@ public void Arrange()
                 }
             };
 
-            _providerRelationsApiClient = new Mock<IProviderRelationshipsApiClient>();
+            _providerRelationsApiClient = new Mock<IReservationsOuterService>();
```

Comment:
rename this parameter to be clearer (ie change from providerRelationsApiClient to _reservationsOuterService